src/index.js

```diff
@@ -2,13 +2,30 @@ const t = require('@babel/types');
 const extractArguments = require('./extractArguments');
 const combineArguments = require('./combineArguments');
 
+const LIBS = ['clsx', 'classnames'];
+
 module.exports = () => {
+  let names = [];
   return {
     visitor: {
+      Program: {
+        enter() {
+          names = [];
```

Comment:
```suggestion
          names = ['clsx', 'classNames']
```
It's not guaranteed that the file contains imports, so it should always contain the defaults.

Reply:
Thank you for your feedback, defaults sound like a good idea yet these may conflict with other libraries or variables plugin uses may use. Perhaps these can be the defaults which users specify in plugin settings. What do you think?

Reply:
> Perhaps these can be the defaults which users specify in plugin settings. What do you think?

A setting that overrides the defaults would be the way to go.

Reply:
Definitely gonna add the user options, yet not sure about `defaultNames` as these still may conflict with other function names. I believe a better option is to not to modify code if we are not sure whether it is `classnames` or `clsx`.

Reply:
> yet not sure about defaultNames as these still may conflict with other function names

Then i'd rather have the user disable it, the plugin should do it's job out of the box.
If they have some setup that causes issues with this, they can disable it.